dockers/docker-bgp/render_config.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+import sys
+import os.path
+import argparse
+
+from minigraph_facts import parse_xml
+import jinja2
+import netaddr
+
+
+def is_ipv4(value):
+    if not value:
+        return False
+    if isinstance(value, netaddr.IPAddress):
+        addr = value
+    else:
+        try:
+            addr = netaddr.IPAddress(str(value))
+        except:
+            return False
+    return addr.version == 4
+
+def is_ipv6(value):
+    if not value:
+        return False
+    if isinstance(value, netaddr.IPAddress):
+        addr = value
+    else:
+        try:
+            addr = netaddr.IPAddress(str(value))
+        except:
+            return False
+    return addr.version == 6
+
+
+def main():
+    parser=argparse.ArgumentParser(description="Render configuration file from minigraph data and jinja2 template.")
+    parser.add_argument("template")
+    parser.add_argument("-m", "--minigraph", required=True, help="minigraph xml file")
+    #parser.add_argument("-v", "--variable-file", help="yaml file that contains addtional variables")
+    args = parser.parse_args()
+
+    minigraph = args.minigraph
+    template_file = args.template
+    
+    data = parse_xml(minigraph)
+
+    env = jinja2.Environment(loader=jinja2.FileSystemLoader('./'), trim_blocks=True)
+    env.filters['ipv4'] = is_ipv4
```

Comment:
>filters [](start = 8, length = 7)

How many kinds of filters will we use? If many, we should prevent consider implementation and maintenance cost. One option is to reuse ansible implementation.